docker/Dockerfile.crontabber

```diff
@@ -1,5 +1,18 @@
 FROM local/socorro_base
 
+# Install nodejs and npm from Nodesource's 8.x branch
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y curl
+RUN curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - && \
+    echo 'deb https://deb.nodesource.com/node_8.x jessie main' > /etc/apt/sources.list.d/nodesource.list && \
+    echo 'deb-src https://deb.nodesource.com/node_8.x jessie main' >> /etc/apt/sources.list.d/nodesource.list
+RUN DEBIAN_FRONTEND=noninteractive apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y nodejs
+
+# Install developer dependencies from the webapp
+COPY ./webapp-django/package.json /webapp-frontend-deps/package.json
+COPY ./webapp-django/package-lock.json /webapp-frontend-deps/package-lock.json
+RUN cd /webapp-frontend-deps/ && npm install --only=dev
+
```

Comment:
This is all repeated from `Dockerfile.webapp`. Maybe it's a better idea to have `Dockerfile.crontabber` derive from `local/socorro_webapp` instead of `local/socorro_base`? Either that or move most of these lines (everything except the `COPY` ones) into a `setup_node_deps.sh` script which gets run in both `Dockerfile.webapp` and `Dockerfile.crontabber`?